pkg/scanners/trivy/types.go

```diff
@@ -83,8 +82,11 @@ func (s *ImageScanner) Scan(img eraserv1alpha1.Image) (ScanStatus, error) {
 			continue
 		}
 
+		imageScanContext, cancel := context.WithTimeout(context.Background(), *imageScanTimeout)
+		defer cancel()
+
 		scanner := scanner.NewScanner(s.scanConfig.localScanner, artifactToScan)
-		report, err := scanner.ScanArtifact(s.ctx, s.scanConfig.scanOptions)
+		report, err := scanner.ScanArtifact(imageScanContext, s.scanConfig.scanOptions)
```

Comment:
What does the error message look like when this times out? I want us to make sure it's clear to the user why the scan failed.